server/utils/licenceComparator.test.ts

```diff
@@ -467,7 +440,7 @@ describe('Licence Comparator', () => {
       licenceConditionsRemoved: [
         {
           category: 'Freedom of movement',
-          condition: 'Exclusion zones have been removed',
```

Comment:
This is the only substantial (ie spec-impacting) change of this file, the others are only because `uploadSummary` is no longer used in this method